providers/burst-sms/src/lib/burst-sms.provider.ts

```diff
@@ -14,8 +14,8 @@ export class BurstSmsProvider implements ISmsProvider {
 
   constructor(
     private config: {
-      apiKey: string;
-      secretKey: string;
+      apiKey?: string;
+      secretKey?: string;
```

Comment:
I do think we have an implementation problem if we need to make optional the keys for authenticating in the providers. At this point we should have already checked we have the values set.

Reply:
This is because the interface for the `ICredentials` has these fields as optional, I didn't want to touch that as it might require even more work to fix.